src/devices/src/legacy/rtc_pl031.rs

```diff
@@ -171,26 +171,20 @@ impl BusDevice for RTC {
                 }
             }
         } else {
-            warn!(
-                "Invalid RTC PL031 data length {}",
-                data.len()
-            );
+            warn!("Invalid RTC PL031 data length {}", data.len());
```

Comment:
Would it be worth it to state that this is a `read` warning in the message? Same for the `write`.

Reply:
Good idea! I made the changes